src/isValid.js

```diff
@@ -1,8 +1,7 @@
-'use strict';
-import curry from 'lodash.curry';
+import curry from 'lodash.curry'
 
 export default curry((date) => {
-    if(Object.prototype.toString.call(date) !== "[object Date]") return false;
-    let unixTime = date.getTime();
-    return unixTime === unixTime;
-});
+  if (Object.prototype.toString.call(date) !== "[object Date]") return false
+
+  return !isNaN(date.getTime())
```

Comment:
Think that would have to be !== because `NaN` is the only value that doesn't equal itself.